### PR TITLE
[DOC] Add ingestors block to tanka doc

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:
Adds a configuration option for Tanka to reduce ingestor resource requirements.

### CODE REVIEW

File: docs/tempo/website/setup/tanka.md [hunk 1]
```diff
@@ -339,6 +339,31 @@ storage+: {
...
+    resources+: {
+        limits+: {
+            cpu: '3',
+            memory: '5Gi',
+        },
+        requests+: {
+            memory: '2Gi',
+            cpu: '200m',
+        },
+    },
```
Comment (knylander-grafana) [inline docs/tempo/website/setup/tanka.md:362]:
```suggestion
      },
```

**Review by hedss — APPROVED** [review]
LGTM!

**Review by zalegrala — APPROVED** [review]

File: docs/tempo/website/setup/tanka.md [hunk 2]
```diff
@@ -339,6 +339,31 @@ storage+: {
...
+        limits+: {
+            cpu: '3',
+            memory: '5Gi',
+        },
+        requests+: {
+            memory: '2Gi',
+            cpu: '200m',
+        },
+      },
+   }
```
Comment (knylander-grafana) [inline docs/tempo/website/setup/tanka.md:363]:
```suggestion
     },
    },
      tempo_distributor_container+:: {
        securityContext+: {
          runAsUser: 0,
     },
        resources+: {
          limits+: {
          cpu: '3',
          memory: '5Gi',
      },
        requests+: {
          memory: '2Gi',
          cpu: '200m',
      },
    },
  },
```

File: docs/tempo/website/setup/tanka.md [hunk 3]
```diff
@@ -339,6 +339,31 @@ storage+: {
...
+    resources+: {
+        limits+: {
+            cpu: '3',
+            memory: '5Gi',
+        },
+        requests+: {
+            memory: '2Gi',
+            cpu: '200m',
+        },
+      },
```
Comment (knylander-grafana) [inline docs/tempo/website/setup/tanka.md:362]:
```suggestion
    tempo_ingester_container+:: {
       securityContext+: {
          runAsUser: 0,
       },
       resources+: {
        limits+: {
            cpu: '3',
            memory: '5Gi',
        },
        requests+: {
            memory: '2Gi',
            cpu: '200m',
        },
     },
```

**Comment by knylander-grafana** [comment]
@hedss  Do we want the securitycontext line in the config file block?

**Comment by knylander-grafana** [comment]
Per a thread with @danstadler-pdx, I added a line in the config block for securityContext. I'm waiting on verification that this line is correct prior to merging this PR.

**Pull request merged**